scripts/tendermint/all_start.sh

```diff
@@ -5,6 +5,7 @@ command -v shellcheck > /dev/null && shellcheck "$0"
 declare -a TM_VERSIONS
 TM_VERSIONS[25]=0.25.0
 TM_VERSIONS[27]=0.27.4
+TM_VERSIONS[29]=0.29.1
```

Comment:
The same line must be added in all_start.sh as well

Reply:
done